### PR TITLE
Minor fixes & improvements

### PR DESCRIPTION
**Pull request by temocj**
Hello!

This pull request fixes some errors as of Swift 5.3 and adds a few miscellaneous examples.

This is my first pull request (ever). I hope I did everything right!

Do you like the changes?

🙏

### CODE REVIEW

**Comment by Kapeli** [comment]
Can anyone review these changes? Thank you!

// cc @jbcrail @Lumus @PrestonKnopp @marlonandrade @henneonrails @varokas

File: cheatsheets/Swift.rb [hunk 1]
```diff
@@ -28,20 +28,36 @@ class MyClass : OptionalSuperClass, OptionalProtocol {
       notes <<-'END'
         ``` swift
         class MyClass {
+            // Class Methods
+            static func course() -> String { return "Swift!" }
```
Comment (Lumus) [inline cheatsheets/Swift.rb:32]: For this to be a "Class Method" it should be `class func` not `static func`. Typically `static func` should be used for value types and `class func` should be used for reference types.

Additionally I would consider removing `return` (not just here but wherever possible) to highlight cases where it is not necessary.
```suggestion
            static func course() -> String { "Swift!" }
```

File: cheatsheets/Swift.rb [hunk 2]
```diff
@@ -78,7 +95,7 @@ class MyClass {
             }
             didSet { // never set greater than 10
```
Comment (Lumus) [inline cheatsheets/Swift.rb:96]: For clarity I'd update the comment to reflect what's happening
```suggestion
            didSet { // ignore new value if greater than 10
```

File: cheatsheets/Swift.rb [hunk 3]
```diff
@@ -344,8 +368,8 @@ class MyClass : MyProtocol {
       name 'Error Handling'
       notes <<-'END'
         ``` swift
-        // ErrorType
-        enum MyError : ErrorType {
+        // Error
+        enum MyError : Error {
```
Comment (Lumus) [inline cheatsheets/Swift.rb:372]: I'd consider conforming to `LocalizedError` and implementing `var errorDescription: String?` Something like: 
```swift
enum MyError: LocalizedError {
    case Err1
    case Err2(errDesc: String)
    
    var errorDescription: String? {
    switch self {
        case .Err1: "Error 1 occurred"
        case let .Err2(errDesc): "Error 2 occurred: \(errDesc)"
        }
    }
}

Then update line 397 to:

```swift
print(error.localizedDescription)
```

**Review by Lumus — APPROVED** [review]

**Comment by temocj** [comment]
Thanks, Lumus!

Anything else?

**Comment by Lumus** [comment]
Nope, I think it looks good 👍

**Comment by temocj** [comment]
Cool, thanks for your review & approval.

**Comment by Kapeli** [comment]
Thank you @temocj and @Lumus!

**Pull request merged**